www/src/pages/guidelines/typography.js

```diff
@@ -0,0 +1,600 @@
+          <Box maxWidth="35rem">
+            <Text
+              as="p"
+              fontFamily="header"
+              fontWeight={0}
+              fontSize={4}
+              letterSpacing="tracked"
+              css={{
+                textTransform: `uppercase`,
+              }}
+            >
+              The Future is Fast
+            </Text>
+            <Heading
+              fontSize={{ xxs: 8, sm: 12, lg: 13 }}
+              letterSpacing="tight"
+              lineHeight="solid"
+              mb={5}
+            >
+              Create digital experiences on the edge—faster
+            </Heading>
+            <Heading as="h2" mb={5} fontSize={6} fontWeight={1}>
+              Gatsby provides a modern framework for turning content into
+              feature-rich, visually engaging apps and websites.
+            </Heading>
+            <Text fontSize={{ xxs: 2, md: 3 }} mb={7}>
+              <p>
+                Over the last few years, the modern Javascript ecosystem has
+                created tools that allow developers to build quicker with fewer
+                bugs. Gatsby gives you easy access to features like modern
+                Javascript syntax, code bundling and hot reloading, without
+                having to maintain custom tooling. Build app-like experiences
+                faster — with Gatsby.
+              </p>
+              <p>
+                Websites come in a thousand different flavors. Timeframes,
+                budgets, interactivity requirements and content systems can vary
+                wildly from one project to the next.
+              </p>
+              <p>
+                This variety puts website teams between a rock and a hard place.
+                They often have to maintain frontends built in multiple
+                development systems, stretching their developers’ skill sets.
+                Implementing the same dropdown in five different frameworks can
+                be a huge headache. But what’s the alternative — turn down good
+                client projects?
+              </p>
+              <p>
+                To add to the difficulty, when your UI development framework is
+                coupled to your client’s CMS backend, it doesn’t just cause
+                technical problems; it causes people problems. It makes your
+                team’s staffing plans dependent on specific projects. It
+                hamstrings your ability to respond to changing client
+                requirements by shifting resources around.
+              </p>
+            </Text>
+            <Button bg="black">Read more</Button>
+          </Box>
+        </ContentColumn>
+      </Columns>
+    </Section>
+
+    <ThemeProvider theme={themeInverted}>
+      <Container
+        bg="white"
+        py={8}
+        my={8}
+        textAlign="center"
+        css={{ WebkitFontSmoothing: `antialiased` }}
+      >
+        <Text fontSize={1} color="blackFade.60">
```

Comment:
This text is pretty difficult to read as well (grey on black)

Reply:
I changed this to `grey.90`, for which I get a contrast ratio of 11.12. Would this be OK?

Reply:
Nevermind, decided to remove the yellow background!